admin-dev/themes/new-theme/scss/pages/product/create_product_default_theme.scss

```
@@ -0,0 +1,17 @@
+#modal-create-product {
+  //modal loader
+  @import "~bootstrap/scss/mixins/border-radius";
+  @import "../../config/settings";
+  @import "~prestakit/scss/spinners";
+  @import "../../components/modal";
+  @import "create_product_modal";
+
+  .d-none {
+    display: none !important;
```

Seems like this line is bothering the scss linter because of the `!important`, so you need to ass an ignore comment exceptionally

```suggestion
    /* stylelint-disable declaration-no-important */
    display: none !important;
    /* stylelint-enable declaration-no-important */
```